src/physics/em/BetheHeitlerInteractor.hh

```diff
@@ -0,0 +1,132 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BetheHeitlerInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+#include "physics/base/Interaction.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/Secondary.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/base/Units.hh"
+#include "physics/material/ElementView.hh"
+#include "BetheHeitlerInteractorPointers.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Bethe-Heitler model for gamma -> e+e- (electron-pair production).
+ *
+ * Give an incident gamma, it adds a two pair-produced secondary electrons to
+ * the secondary stack. No cutoffs are performed on the incident gamma energy.
+ *
+ * \note This performs the same sampling routine as in Geant4's
+ *  G4BetheHeitlerModel, as documented in section 6.5 of the Geant4 Physics
+ *  Reference (release 10.6), applicable to incident gammas with energy
+ *  E_gamma \leq 100 GeV. For E_gamma > 80 GeV, it is suggested to use
+ *  `G4PairProductionRelModel`.
+ */
+class BetheHeitlerInteractor
+{
+  public:
+    //! Construct sampler from shared and state data
+    //! TODO: Handle Material through `shared`?
```

Comment:
I think the TODO can be removed. Also as a minor comment, the special doxygen `//!` should be attached to the function definition (which it is) so just remove the `!` here.

Reply:
Removed. (I think I prematurely removed the "WIP" from this.)